NOnion/Constants.fs

```diff
@@ -18,3 +18,61 @@ module Constants =
     (* Amount of bytes needed for generating keys and digest during KDF = 2 * KeyLength + 3 * HashLength *)
     [<Literal>]
     let KdfLength = 92
+
```

Comment:
i guess Constants module should be internal?

Reply:
We use them in test projects and since it's not internal right now, let's not change that in this PR

Reply:
then let's mark as internal the elements that are only there to avoid magic numbers

Reply:
not addressed yet